.github/workflows/build_wheels.yml

```diff
@@ -249,8 +254,8 @@ jobs:
         CIBW_BEFORE_BUILD: pip install cython && pip install --upgrade setuptools
       run: cibuildwheel --print-build-identifiers && cibuildwheel --output-dir wheelhouse
       shell: bash
-    - name: install sha512sum on MacOS
-      if: startsWith(matrix.os_python.os, 'macos')
+    - name: Install sha512sum on MacOS
+      if: contains(matrix.os_python.os, 'macos-latest')
```

Comment:
Any reason for this change? I think I like the previous version more because it is permissive if we eventually decide to pin to a specific macos version

Reply:
I think this test isn't quite right - https://github.com/sre-apache-beam-dev/beam/actions/runs/3041505275/workflow#L258 - what I'm suggesting is that we keep `startsWith(matrix.os_python.os, 'macos')` - that tests `contains(matrix.os_python.os, 'macos')`

With that said, because we're using arrays for the other cases I'm not sure how GHA will handle it (it'll try to cast them to strings, but that might blow up?). I'm fine leaving it as is